src/access/RequestPath.h

```diff
@@ -26,8 +26,10 @@ namespace Access {
 struct RequestPath
 {
     // NOTE: eventually this will likely also contain node, for proxying
-    ClusterId cluster   = 0;
-    EndpointId endpoint = 0;
+    ClusterId cluster     = 0;
+    EndpointId endpoint   = 0;
+    uint32_t entityId     = 0; // attribute, command, or event id.  Ignored if entityWildcarded is true
+    bool entityWildcarded = false;
```

Comment:
entityWildcarded seems to be entirely unused (looking at the everything-together PR at https://github.com/project-chip/connectedhomeip/pull/34485.  Why is it here?

Reply:
Oops, missed it in the PR breakup. Good catch.

Reply:
Fixed.